pandas/tests/groupby/test_transform.py

```diff
@@ -789,7 +789,7 @@ def test_pct_change(test_series, freq, periods, fill_method, limit):
     df = DataFrame({'key': key_v, 'vals': vals * 2})
 
     df_g = getattr(df.groupby('key'), fill_method)(limit=limit)
-    grp = df_g.groupby('key')
+    grp = df_g.groupby(df.key)
```

Comment:
can you add another example using 'key' as well here

Reply:
not sure what you're looking for here? `df_g` no longer has a column `'key'`, and this section of code is only used to compute `expected`, it's not what's being tested

Reply:
why did u have to change it?is the expected difference for these cases?

Reply:
The old test called something like `df.groupy('key').ffill().groupby('key').shift()`, which is no longer valid, as the first groupby won't return the `'key'` column. The expected is still the same, just needs to be written differently.